tests/std/tests/Dev09_158457_tr1_mem_fn_calling_conventions/test.cpp

```diff
@@ -64,7 +64,7 @@ struct Cat {
     }
 #endif
 
-#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE)
+#if (defined(_M_IX86) || (defined(_M_X64) && !defined(_M_ARM64EC))) && !defined(_M_CEE)
     int __vectorcall g() {

```

Comment:
No change requested: This test has "expected results" below that would need to be updated if we ever add runtime coverage of ARM64EC, but we can do that if/when that happens.